templates/shared/forms/interactive/automotive.html

```diff
@@ -0,0 +1,78 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <div class="js-pagination js-pagination--1" style="padding-top:2rem;">
```

Comment:
I don't know that this is necessarily an issue as the form closes just fine, but I've not seen an interactive form with the close button on the bottom as opposed to the top before. Normally there's a header section (example [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/ab7842dfea5ffa5bf0f894a999c299bba2730d56/templates/shared/forms/interactive/lxd.html#L3)), is this a new design pattern?

Reply:
The request from the pdm was for no header content. Without any content it looked a bit off as there was a floating line at the top and this overlapped with the `X`. But I am not sure if this is the best way to handle this.